src/views/RecIM/components/List/Listing/index.js

```diff
@@ -61,23 +64,26 @@ const ParticipantListing = ({ username }) => {
             variant="rounded"
           ></Avatar>
         </ListItemAvatar>
-        <Link to={`/profile/${username}`} className="gc360_link">
-          <ListItemText primary={username} />
+        <Link to={`/profile/${participant.username}`} className="gc360_link">
+          <ListItemText primary={participant.username} />
         </Link>
       </Grid>
     </ListItem>
   );
 };
 
-const MatchListing = ({ activityID, matchID }) => {
+const MatchListing = ({ match }) => {
   return (
-    <>
-      <Link to={`/recim/activity/${activityID}/match/${matchID}`} className="gc360_link">
-        <Grid container className={styles.listing}>
-          <Grid item>Team A vs Team B</Grid>
-        </Grid>
-      </Link>
-    </>
+    <ListItem
+      button
+      component={Link}
+      to={`/recim/activity/${match.activityID}/match/${match.ID}`}
+      className="gc360_link"
+    >
+      <Grid container className={styles.listing}>
+        <Grid item>Team A vs Team B</Grid>
+      </Grid>
+    </ListItem>
```

Comment:
Next PR should have some of these details fleshed out.
i.e.
Team A vs Team B can already be detailed to Match.Team[0].Name vs Match.Team[1].Name for any match instance that is not Ladder styled. You could probably even handle this with a foreach with a catch the last element in the array.